ui/src/components/issuers.tsx/Issuers.tsx

```diff
@@ -0,0 +1,89 @@
+import { Divider, Modal, Space, message } from "antd";
+import { useCallback, useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { SiderLayoutContent } from "../shared/SiderLayoutContent";
+
+import { IssuerForm } from "./IssuerForm";
+import { IssuersTable } from "./IssuersTable";
+import { createIssuer } from "src/adapters/api/issuers";
+import IconClose from "src/assets/icons/x.svg?react";
+import { useEnvContext } from "src/contexts/Env";
+import { useIssuerContext } from "src/contexts/Issuer";
+import { IssuerFormData } from "src/domain/identifier";
+import { ROUTES } from "src/routes";
+import { isAsyncTaskDataAvailable } from "src/utils/async";
+import { makeRequestAbortable } from "src/utils/browser";
+import { ISSUERS, ISSUER_ADD } from "src/utils/constants";
+
+export function Issuers() {
+  const env = useEnvContext();
+  const { fetchIssuers, issuersList } = useIssuerContext();
+
+  const [messageAPI, messageContext] = message.useMessage();
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
Why do we have a modal and a route to create issuers?

Reply:
according to the design, if the user does not have issuer, then a modal window will be shown, if there is, then the creation will lead to the page